diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/etc/e-smith/templates/etc/e-smith/web/common/css/sme_core.css/90contribs mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/templates/etc/e-smith/web/common/css/sme_core.css/90contribs
--- smeserver-support-1.6.0/root/etc/e-smith/templates/etc/e-smith/web/common/css/sme_core.css/90contribs	2006-03-16 18:25:05.000000000 +1100
+++ mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/templates/etc/e-smith/web/common/css/sme_core.css/90contribs	2006-07-17 07:11:12.686906311 +1000
@@ -2,7 +2,7 @@
     $OUT =<<'HERE';
 /* contribs.org styling */
 body.header {
-background: #aec0ec;
+background: #bbcbef;
     }
 .hilightbar {
 background-color: #ffffff;
