diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/online-manual mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/online-manual
--- smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/online-manual	2008-02-13 22:00:12.000000000 +1100
+++ mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/online-manual	2008-02-13 21:57:36.000000000 +1100
@@ -19,12 +19,6 @@
 	  ]]>
 	</trans>
       </entry>		
-
-      <entry>
-	<base>Miscellaneous</base>
-	<trans>Miscellaneous</trans>
-      </entry>
-
       <entry>
 	<base>Online manual</base>
 	<trans>Online manual</trans>
diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/support mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/support
--- smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/support	2008-02-13 22:00:12.000000000 +1100
+++ mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/support	2008-02-13 21:57:06.000000000 +1100
@@ -3,12 +3,6 @@
       <base>FORM_TITLE</base>
       <trans>Support, warranty and licensing information</trans>
     </entry>
-
-    <entry>
-        <base>Miscellaneous</base>
-        <trans>Miscellaneous</trans>
-    </entry>
-
     <entry>
         <base>Support and licensing</base>
         <trans>Support and licensing</trans>
