diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/initial.cgi mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/initial.cgi
--- smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/initial.cgi	2005-09-17 14:59:08.000000000 +1000
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/initial.cgi	1970-01-01 10:00:00.000000000 +1000
@@ -1,62 +0,0 @@























-        <br>


-	<br>
-      Englisch: <A HREF="http://www.contribs.org/" TARGET="http://contribs.org">contribs.org  </A>
-	<br>


-	<br>



-	<br>
-	<br>

























diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/support mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/support
--- smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/support	2005-09-17 07:18:48.000000000 +1000
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/de/etc/e-smith/web/functions/support	1970-01-01 10:00:00.000000000 +1000
@@ -1,18 +0,0 @@


















diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/initial.cgi mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/initial.cgi
--- smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/initial.cgi	2005-07-18 17:00:51.000000000 +1000
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/initial.cgi	1970-01-01 10:00:00.000000000 +1000
@@ -1,55 +0,0 @@























































diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/support mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/support
--- smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/support	2003-03-29 01:20:56.000000000 +1100
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/support	1970-01-01 10:00:00.000000000 +1000
@@ -1,17 +0,0 @@

















diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi
--- smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi	2005-10-07 11:35:41.000000000 +1000
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi	1970-01-01 10:00:00.000000000 +1000
@@ -1,45 +0,0 @@
-<lexicon lang="it">
-    <entry>
-	<base>FORM_TITLE</base>
-	<trans>Benvenuto nel server manager</trans>
-    </entry>
-
-    <entry>
-	<base>FRAMES_BODY</base>
-	<trans>
-	<![CDATA[
-	<p>
-	Questa è la unsupported developer release del Mitel Networks SME
-	Server. Questo software non ha NESSUNA GARANZIA. Fare clic
-	<A HREF="cgi-bin/support/" TARGET="main">qui</A>
-	per avere informazioni dettagliate su supporto, garanzia e licenza.
-	</p>
-
-	<p>
-	Per amministrare il sistema, fare clic su uno dei link nel menu alla
-	sinistra dello schermo.
-	</p>
-	]]>
-	</trans>
-    </entry>
-
-    <entry>
-	<base>NOFRAMES_BODY</base>
-	<trans>
-	<![CDATA[
-	<p>
-	Questa è la unsupported developer release del Mitel Networks SME
-	Server. Questo software non ha NESSUNA GARANZIA. Fare clic
-	<A HREF="cgi-bin/support/" TARGET="main">qui</A>
-	per avere informazioni dettagliate su supporto, garanzia e licenza.
-	</p>
-
-	<p>
-	Fare clic
-	<A HREF="/common/noframes">qui</A> per una
-	lista delle funzioni di amministrazione.
-	</p>
-	]]>
-	</trans>
-    </entry>
-</lexicon>
diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support
--- smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support	2005-10-07 11:35:41.000000000 +1000
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support	1970-01-01 10:00:00.000000000 +1000
@@ -1,17 +0,0 @@
-<lexicon lang="it">
-    <entry>
-      <base>FORM_TITLE</base>
-      <trans>Informazioni sul supporto, la garanzia e la licenza</trans>
-    </entry>
-
-    <entry>
-      <base>Miscellaneous</base>
-      <trans>Varie</trans>
-    </entry>
-
-    <entry>
-      <base>Support and licensing</base>
-      <trans>Supporto e licenza</trans>
-    </entry>
-
-</lexicon>
