diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/createlinks mezzanine_patched_smeserver-support-1.4.7/createlinks
--- smeserver-support-1.4.7/createlinks	2005-11-18 17:47:56.922110471 +1100
+++ mezzanine_patched_smeserver-support-1.4.7/createlinks	2005-11-18 17:47:18.923962246 +1100
@@ -9,6 +9,6 @@
 
 panel_link("support", $panel);
 
-templates2events("/etc/$_", "bootstrap-console-save") 
+templates2events("/etc/$_", qw(post-install post-upgrade))
 	for (qw(issue issue.net motd redhat-release));
 
