diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi mezzanine_patched_smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi
--- smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/initial.cgi	2005-09-30 06:07:15.579139408 -0600
@@ -0,0 +1,45 @@
+<lexicon lang="it">
+    <entry>
+	<base>FORM_TITLE</base>
+	<trans>Benvenuto nel server manager</trans>
+    </entry>
+
+    <entry>
+	<base>FRAMES_BODY</base>
+	<trans>
+	<![CDATA[
+	<p>
+	Questa è la unsupported developer release del Mitel Networks SME
+	Server. Questo software non ha NESSUNA GARANZIA. Fare clic
+	<A HREF="cgi-bin/support/" TARGET="main">qui</A>
+	per avere informazioni dettagliate su supporto, garanzia e licenza.
+	</p>
+
+	<p>
+	Per amministrare il sistema, fare clic su uno dei link nel menu alla
+	sinistra dello schermo.
+	</p>
+	]]>
+	</trans>
+    </entry>
+
+    <entry>
+	<base>NOFRAMES_BODY</base>
+	<trans>
+	<![CDATA[
+	<p>
+	Questa è la unsupported developer release del Mitel Networks SME
+	Server. Questo software non ha NESSUNA GARANZIA. Fare clic
+	<A HREF="cgi-bin/support/" TARGET="main">qui</A>
+	per avere informazioni dettagliate su supporto, garanzia e licenza.
+	</p>
+
+	<p>
+	Fare clic
+	<A HREF="/common/noframes">qui</A> per una
+	lista delle funzioni di amministrazione.
+	</p>
+	]]>
+	</trans>
+    </entry>
+</lexicon>
diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support mezzanine_patched_smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support
--- smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-support-1.4.6/root/etc/e-smith/locale/it/etc/e-smith/web/functions/support	2005-09-30 06:07:14.215346736 -0600
@@ -0,0 +1,17 @@

















