diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/createlinks mezzanine_patched_smeserver-support-1.6.0/createlinks
--- smeserver-support-1.6.0/createlinks	2006-01-04 22:58:14.000000000 -0700
+++ mezzanine_patched_smeserver-support-1.6.0/createlinks	2007-06-06 16:47:10.000000000 -0600
@@ -12,3 +12,4 @@
 templates2events("/etc/$_", qw(post-install post-upgrade))
-	for (qw(issue issue.net motd redhat-release));
+	for (qw(issue issue.net motd redhat-release sysconfig/hw-uuid));
 
+service_link_enhanced("smolt", "S90", "7");
diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/status mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/status
--- smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/status	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/status	2007-06-06 16:44:17.000000000 -0600
@@ -0,0 +1 @@
+enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/type mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/type
--- smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/type	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/db/configuration/defaults/smolt/type	2007-06-06 16:44:17.000000000 -0600
@@ -0,0 +1 @@
+service
diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/etc/e-smith/templates/etc/sysconfig/hw-uuid/template-begin mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/templates/etc/sysconfig/hw-uuid/template-begin
--- smeserver-support-1.6.0/root/etc/e-smith/templates/etc/sysconfig/hw-uuid/template-begin	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-support-1.6.0/root/etc/e-smith/templates/etc/sysconfig/hw-uuid/template-begin	2007-06-06 17:36:10.000000000 -0600
@@ -0,0 +1 @@
+{ use Data::UUID; $sysconfig{SystemID} || new Data::UUID->create_str }
