diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue
--- smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue	2005-11-18 14:18:54.480838575 +1100
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-TEMPLATE_PATH="/etc/motd"
diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue.net mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue.net
--- smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue.net	2005-11-18 14:18:54.481838402 +1100
+++ mezzanine_patched_smeserver-support-1.4.7/root/etc/e-smith/templates.metadata/etc/issue.net	2005-11-18 14:18:24.237089190 +1100
@@ -1 +1 @@
-TEMPLATE_PATH="/etc/motd"
+TEMPLATE_PATH="/etc/issue"
