diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl mezzanine_patched_smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl
--- smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl	2008-03-08 16:26:13.000000000 +1100
+++ mezzanine_patched_smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl	2008-03-08 16:24:38.000000000 +1100
@@ -61,7 +61,7 @@
     {
         ($rc, $choice) = $console->message_page
             (
-             title => "Internet connection failed",
+             title => gettext("Internet connection failed"),
              text  =>
              gettext("The test failed. Your server was unable to establish contact with contribs.org via the Internet.") .
              "\n\n" .
