diff -Nur -x '*.orig' -x '*.rej' smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl mezzanine_patched_smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl
--- smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl	2008-03-18 16:08:23.000000000 -0600
+++ mezzanine_patched_smeserver-support-1.6.0/root/sbin/e-smith/console-menu-items/testInternet.pl	2008-03-18 16:07:55.000000000 -0600
@@ -7,7 +7,7 @@
 {
     my $class = shift;
     my $self = {
-		    name => "Test Internet access",
+		    name => gettext("Test Internet access"),
 		    order => 30,
 		};
     bless $self, $class;
